src/components/SlateEditor/EditorFooter.tsx

```diff
@@ -152,7 +162,7 @@ const EditorFooter: React.FC<Props> = ({
           </FooterLinkButton>
         )}
         <StyledLine />
-        {values.id && (
+        {values.id && !isConceptType() && (
```

Comment:
Valideringa skal fortsette bare i frontenden? for vi har ikke noe validerings endepunkt i concept-api

Reply:
Klarer oss med validering i frontend enn så lenge.